prototypes/index.js

```diff
@@ -297,6 +298,71 @@ const classPrompts = {
 
 
 
+// ---------------------------------------------------------------------------
+// ---------------------------------------------------------------------------
+// ---------------------------------------------------------------------------
+// ---------------------------------------------------------------------------
+// ---------------------------------------------------------------------------
+
+
+// DATASET: nationalParks from ./datasets/nationalParks
+
+const nationalParksPrompts = {
+  getParkVisitList() {
+    /// Return an object containing the names of which parks I need to visit
+    // and the ones I have already visited eg:
+    // {
+    //   parksToVist: ["Yellowstone", "Glacier", "Everglades"],
+    //   parksVisited: ["Rocky Mountain", "Acadia", "Zion"]
+    //}
+
+    const result = 'REPLACE WITH YOUR RESULT HERE';
+    return result;
+
+    // Annotation:
+    // Write your annotation here as a comment
+  },
+
+  getParkInEachState() {
```

Comment:
Love this prompt, think it will reinforce some use cases for bracket notation students don't often see